modules/rosa-sts-aws-requirements.adoc

```diff
@@ -0,0 +1,43 @@
+[id="rosa-sts-customer-requirements_{context}"]
+= Customer Requirements
+
+
+{product-title} (ROSA) clusters must meet several prerequisites before they can be deployed.
+
+[NOTE]
+====
+In order to create the cluster, the user must be logged in as an IAM user and not an assumed role or STS user.
```

Comment:
@jharrington22 @iamkirkbater is this still true? I thought we addressed feedback from Consulting team and enabled an AssumeRole user with STS to initiate the creation process?

Reply:
This is not true any more, users can have STS credentials to create STS clusters.

Reply:
@billmvt  @vkareh  I will delete the Note. If you think we need an different note, please provide info. thx!